modules/EmailTemplates/EmailTemplate.php

```diff
@@ -835,7 +648,7 @@ public function bean_implements($interface)
 
     public static function getTypeOptionsForSearch()
     {
-        $template = BeanFactory::newBean('EmailTemplates');
+        $template = new self();
```

Comment:
Reason for this change? even though we are in the same class I would lean towards using the standard BeanFactory if we can. Never know, we might one day deprecate this whole class and have all the ```BeanFactory ``` calls to ```EmailTemplates``` point somewhere else.